pwiz_tools/Skyline/Model/ElementLocators/ExportAnnotations/ElementHandler.cs

```diff
@@ -86,6 +106,19 @@ public static IList<ElementHandler> GetElementHandlers(SkylineDataSchema dataSch
                     new TransitionResultHandler(dataSchema)
             };
         }
+
+        /// <summary>
+        /// Retrieve a list of all possible Element Handlers
+        /// </summary>
+        /// <returns>A list of Element handlers</returns>
+        public static IList<ElementHandler> GetAllHandlers()
```

Comment:
I would not expect the ElementHandler's that are returned here to work correctly in terms of handling annotations.
The dictionary "_annotationDefs" gets populated in the ElementHandler constructor by looking at the SrmDocument in the DataSchema.
If you use a blank document, then you will get an ElementHandler that does not know about any of the annotations that are enabled in the document.

Reply:
Ok, I tried to address this by moving the method to CommandLine.cs and using the actual document instead of a blank document. 
I kept the original method so that CommandArgs.cs has a static method it can use to access a list of ElementHandler names to display to the user (without actually using them to handle annotations). Could you let me know what you think?